packages/mobile/src/screens/edit-track-screen/components/index.ts

```diff
@@ -1,5 +1,5 @@
 export * from './CancelEditTrackDrawer'
 export * from './RemixTrackPill'
-export * from '../fields/SpecialAccessRadioField'
-export * from '../fields/GollectibleGatedRadioField'
+export * from '../../../components/edit/PriceAndAudienceField/SpecialAccessRadioField'
+export * from '../../../components/edit/PriceAndAudienceField/GollectibleGatedRadioField'
```

Comment:
seems a bit unusual to export from two layers up

Reply:
oh yeah good point, these are not needed, just in renames this carried over, removing!